ppafm/cli/generateDFTD3.py

```diff
@@ -22,6 +22,7 @@ def main():
 
     parser.add_argument('-i', '--input',       action='store',
         help='Input file. Mandatory. Supported formats are: .xyz, .cube, .xsf.')
+    parser.add_argument("--format", action="store", type="string", help="Format of the input geometry file (overrides format concluded from the file name extension)", default=None)
     parser.add_argument('-f', '--data_format', action='store', default='xsf',
```

Comment:
I think it would better to reconsider the names of the arguments. The `format` and `data_format` may be confusing for a new user. Maybe `input_format` and `output_format` instead.

But I think the CLI arguments need an overhaul anyway, so this could be done separately. Also related to #148.

Reply:
I'm working on overhauling the CLI arguments as a part of resolving #148, so let's not touch this here.